docker-compose.dev.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.8"
```

Comment:
besides this line, i think we can remove all other double quotes in this line and everything should still work

Reply:
Done